### PR TITLE
Adjustment to configuration Screen

### PR DESCRIPTION
**Pull request by BKeyport**
In reference to issue #211 I've re-wrote the installation introduction page - including a couple of examples of more complex installation examples. Also, added a link to the forums at the bottom of the page.

### CODE REVIEW

File: configuration/introduction.md [hunk 1]
```diff
@@ -1,47 +1,97 @@
...
+| MM_CONFIG_FILE            | This specifies an alternate configuration file for the system. This is useful when running multiple mirrors on the same device. This does not work with the template option below |
+| MM_PORT                   | This specifies an alternate TCPIP port, overriding "port" item within the config file. This is useful for testing to see if the product will run using another port. |
+
+##### Examples of use: 
+
+You have a second configuration file set up called `config2.js`, create a bash script called `mm2.js`:
+
+```bash
+cd ~/MagicMirror
+export MM_CONFIG_FILE=${pwd}/config/config2.js
```
Comment (khassel) [inline configuration/introduction.md:79]: export MM_CONFIG_FILE=$(pwd)/config/config2.js

File: configuration/introduction.md [hunk 2]
```diff
@@ -93,15 +143,103 @@ MY_HTTPS=false
...
+```bash
+cd ~/MagicMirror
+npm start
+```
+
+Starting Script 2 (EG: mm2.sh): 
+
+```bash
+cd ~/MagicMirror
+export MM_CONFIG_FILE=${pwd}/config/config2.js
```
Comment (khassel) [inline configuration/introduction.md:189]: export MM_CONFIG_FILE=$(pwd)/config/config2.js

Comment (BKeyport) [reply configuration/introduction.md:189]: The problem is that parens don't seem to work on Pi/bookworm. 

```key1970@officemirror:~/MagicMirror $ export MM_CONFIG_FILE=$(pwd)/config/config2.js
bkey1970@officemirror:~/MagicMirror $ export MM_PORT=8081
bkey1970@officemirror:~/MagicMirror $ DISPLAY=:0 npm start

> magicmirror@2.25.0 start
> DISPLAY="${DISPLAY:=:0}" ./node_modules/.bin/electron js/electron.js

[26.12.2023 14:56.06.740] [LOG]   Starting MagicMirror: v2.25.0
[26.12.2023 14:56.06.749] [LOG]   Loading config ...
[26.12.2023 14:56.06.752] [DEBUG] config template file not exists, no envsubst
[26.12.2023 14:56.06.758] [LOG]   Loading module helpers ...
[26.12.2023 14:56.06.767] [LOG]   Initializing new module helper ...
[26.12.2023 14:56.06.769] [LOG]   Module helper loaded: MMM-EmbedURL
[26.12.2023 14:56.06.769] [LOG]   All module helpers loaded.
[26.12.2023 14:56.06.781] [LOG]   Starting server on port 8081 ...
[26.12.2023 14:56.06.787] [WARN]  You're using a full whitelist configuration to allow for all IPs
[26.12.2023 14:56.06.825] [LOG]   Server started ...
[26.12.2023 14:56.06.827] [LOG]   Connecting socket for: MMM-EmbedURL
[26.12.2023 14:56.06.828] [LOG]   Sockets connected & modules started ...
[26.12.2023 14:56.07.136] [LOG]   Launching application.
```

Results in the defaults.js config taking over and giving me "No Config".

**Comment by BKeyport** [comment]
New testing requires rewrite. Do not merge.

**Pull request closed without merging**